Lib/unittest/mock.py

```diff
@@ -796,8 +798,10 @@ def assert_called_once(_mock_self):
         """
         self = _mock_self
         if not self.call_count == 1:
-            msg = ("Expected '%s' to have been called once. Called %s times." %
-                   (self._mock_name or 'mock', self.call_count))
+            msg = ("Expected '%s' to have been called once. Called %s times.\n"
```

Comment:
The error message is different but very close. I suggest to write an helper method or function to format the exception message.

Reply:
Agreed. Will do and use it to limit the output length for existing code as well.